# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/mozjs/hotfix-icuuc.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- firefox-128.3.1/js/moz.configure.vanilla	2025-04-21 11:52:57.116734420 +0200
+++ firefox-128.3.1/js/moz.configure	2025-04-21 11:53:19.143651835 +0200
@@ -1305,7 +1305,7 @@
     return enable_system_icu
 
 
-system_icu = pkg_check_modules("MOZ_ICU", "icu-i18n >= 73.1", when="--with-system-icu")
+system_icu = pkg_check_modules("MOZ_ICU", "icu-i18n >= 73.1, icu-uc >= 73.1", when="--with-system-icu")
 
 
 @depends(enable_system_icu_option)
